official/PianoDiQualifica/res/sections/unitTests/editor.tex

```diff
@@ -0,0 +1,15 @@
+TU-59 & Verificare che un DSLElement sia descritto da un modello. & N.E. \tabularnewline \hline
+TU-60 & Verificare che un DSLElement contenga delle proprietà grafiche che gli permettono di essere disegnato a video. & N.E. \tabularnewline \hline
+TU-61 & Verificare che la Dashboard venga correttamente creata da un utente. & N.E. \tabularnewline \hline
+TU-62 & Verificare che possano essere aggiunte, rimosse o mudificate le righe della Dashboard. & N.E. \tabularnewline \hline
```

Comment:
`rimosse o mudificate le righe` cambia in `rimosse o modificate le righe` (typo)